### PR TITLE
Add Application class

### PR DESCRIPTION
**Pull request by matiasroldan6**
Fixes #

## Specific Changes

  - 
  -
  -

### CODE REVIEW

File: samples/25.message-reaction/src/main/java/com/microsoft/bot/sample/messagereaction/Application.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+     *     The @Component annotation could be used on the Bot class instead of this method
+     *     with the @Bean annotation.
+     * </p>
+     *
+     * @return The Bot implementation for this application.
+     */
+    @Bean
+    public Bot getBot(ActivityLog log) {
+        return new MessageReactionBot(log);
+    }
```
Comment (Batta32) [inline samples/25.message-reaction/src/main/java/com/microsoft/bot/sample/messagereaction/Application.java:50]: This is not the correct way to do this. 

Check the core-bot for [this](https://github.com/southworks/botbuilder-java/blob/external/feature/southworks/samples/13-core-bot/base/samples/13.core-bot/src/main/java/com/microsoft/bot/sample/core/Application.java#L58)

**Review by Batta32 — CHANGES_REQUESTED** [review]
Apart from the comment of the file. Check that you are missing the [ActivityLog](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/25.message-reaction/Startup.cs#L35) too.

File: samples/25.message-reaction/src/main/java/com/microsoft/bot/sample/messagereaction/Application.java (hunk 1 above)
Comment (matiasroldan6) [reply samples/25.message-reaction/src/main/java/com/microsoft/bot/sample/messagereaction/Application.java:50]: I've been checking this. What you want me is to add a getter that returns the ActivityLog? I checked the [multilingual](https://github.com/southworks/botbuilder-java/blob/main/samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/Application.java#L56) and [qna](https://github.com/southworks/botbuilder-java/blob/main/samples/11.qnamaker/src/main/java/com/microsoft/bot/sample/qnamaker/Application.java#L50) bots and they don't have it. That's the change you want?

Comment (Batta32) [reply samples/25.message-reaction/src/main/java/com/microsoft/bot/sample/messagereaction/Application.java:50]: Exactly, the bots you mentioned don't have the `ActivityLog`. The _similar scenario_ here is the core-bot. Apart from the `getter` method, it also needs to be called in the `getBot` in order to consume the `Activitylog`

**Review by Batta32 — APPROVED** [review]
LGTM

**Pull request merged**